canmatrix/convert.py

```diff
@@ -25,7 +25,8 @@
 
 #from .log import setup_logger, set_log_level
 from canmatrix.log import setup_logger, set_log_level
-logger = setup_logger('root')
+import logging
+logger = logging.getLogger('root')
```

Comment:
Maybe just:

```python3
logger = logging.getLogger(__name__)
```
So this file gets its own logger normally regardless of being imported or run.  Then below we configure the root logger which we don't use directly.

https://docs.python.org/3/library/logging.html#logger-objects